spec/rspec/rails/view_rendering_spec.rb

```diff
@@ -16,7 +16,7 @@ def controller
         end
 
         it "does not render views in a nested group" do
-          expect(group.describe{}.new.render_views?).to be_falsey
+          expect(group.describe {}.new.render_views?).to be_falsey
```

Comment:
It doesn't seem that `{}` is necessary.
Wondering if this changes its semantic in any way, but apparently it doesn't.
Here and two more in this file.